src/compiler/Makefile

```diff
@@ -0,0 +1,5 @@
+all:
```

Comment:
I know, its a paint to create a Makefile for every executable and library, but...  You have to get the config right and doing it in the top-level will drive us all crazy.  So, we've made it so you specify what you are making (an EXE or LIB), set a few options, then include the generic Makefile.incl. Pretty simple, right?